### PR TITLE
Fix TypeError(s) and serialize ExceptionInfo

### LINKED ISSUE #100: v1.0.0-beta Errors
If using the dev version dashboard is flooded with Rollbar's own errors:

`API error: Invalid format. data.body.trace.exception.class is missing and it is required.`

### LINKED ISSUE #106: TypeError
Hi. I run the code from your example and get just such error. I use composer

PHP Fatal error:  Uncaught TypeError: Argument 3 passed to Rollbar\RollbarLogger::log() must be of the type array, null given, called in /home/dev/wwwroot/hotwork/vendor/rollbar/rollbar/src/Rollbar.php on line 57 and defined in /home/dev/wwwroot/hotwork/vendor/rollbar/rollbar/src/RollbarLogger.php:30

### PR DESCRIPTION
**Pull request by MichaelMackus**
Fixes #100, fixes #106

### CODE REVIEW

**Comment by nocive** [comment]
bump

**Comment by MichaelMackus** [comment]
bump @brianr @rokob @jessewgibbs

File: src/DataBuilder.php [hunk 1]
```diff
@@ -307,7 +307,7 @@ protected function getBody($toLog, $context)
         if ($toLog instanceof ErrorWrapper) {
             $content = $this->getErrorTrace($toLog);
         } elseif ($toLog instanceof $baseException) {
-            $content = $this->getExceptionTrace($baseException);
+            $content = $this->getExceptionTrace($toLog);
```
Comment (rokob) [inline src/DataBuilder.php]: 👍

File: src/DataBuilder.php [hunk 2]
```diff
@@ -501,7 +502,7 @@ protected function getRequest()
                 $request->$key = $val;
             }
         }
-        if (is_array($_SESSION) && count($_SESSION) > 0) {
+        if (isset($_SESSION) && is_array($_SESSION) && count($_SESSION) > 0) {
```
Comment (rokob) [inline src/DataBuilder.php]: 👍

**Comment by MichaelMackus** [comment]
Woohoo thanks! :)

**Pull request merged**